care/emr/api/otp_viewsets/login.py

```
@@ -21,11 +22,58 @@
+                )
+            elif purpose == OTPType.reset_password:
+                content = get_sms_content(
+                    settings.OTP_SMS_RESET_PASSWORD_TEMPLATE_PATH,
+                    {"random_otp": random_otp},
+                )
+
+            sms.send_text_message(
+                content=content,
+                recipients=[phone_number],
+            )
+        except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)